framework/src/modules/network/network.js

```diff
@@ -42,6 +44,8 @@ const { Peer } = require('./components/storage/entities');
 
 const hasNamespaceReg = /:/;
 
+const DEFAULT_PEER_BAN_TIME = 86400;
```

Comment:
Shall we get this from the module's config constants?

Reply:
We can do this all at once when tackling this issue https://github.com/LiskHQ/lisk-sdk/issues/3678. I Resolve this conversation for now.

Reply:
Once arrived to this point, we should add this to this file https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/network/defaults/config.js